src/Illuminate/Database/DBAL/TimestampType.php

```diff
@@ -33,13 +36,16 @@ public function getSQLDeclaration(array $column, AbstractPlatform $platform): st
             MySQLPlatform::class,
             MySQL57Platform::class,
             MySQL80Platform::class,
+            MySQL84Platform::class,
```

Comment:
Shouldn't we move these changes to a separate PR?

Reply:
10.x is currently on security only updates.